pkg/kubelet/dockershim/remote/docker_service.go

```diff
@@ -49,6 +49,10 @@ func (d *dockerService) Version(ctx context.Context, r *runtimeApi.VersionReques
 	return d.runtimeService.Version(r.GetVersion())
 }
 
+func (d *dockerService) Status(ctx context.Context, r *runtimeApi.StatusRequest) (*runtimeApi.StatusResponse, error) {
+	return nil, fmt.Errorf("not implemented")
```

Comment:
Shouldn't the kubelet parses out network/runtime status directly from the response of this rpc in syncRuntimeStatus?